utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
I'm not sure I understand what this looks like for the end-user. Can you walk me thru how you'd configure the example where you've got the shimmed app going to stdout and ContainerPilot to stderr? (That was your original suggested use case).